File: 2022/day01/main.py
with open("input.txt") as f:
    data = f.read().split("\n\n")

cals = [sum(int(c) for c in cal) for cal in [s.split() for s in data]]
print(f"Part 1: {max(cals)}")
cals.sort(reverse=True)
print(f"Part 2: {sum(cals[0:3])}")

File: 2022/day01/main.js
const fs = require('fs');
data = fs.readFileSync("input.txt", "utf8");
calories = data.split('\n\n').map(e => e.trim().split('\n')).map(e => e.map(c => parseInt(c))).map(e => e.reduce((a, b) => a + b)).sort((a, b) => b - a);
console.log("Part 1:", calories[0]);
console.log("Part 2:", calories[0] + calories[1] + calories[2]);
